v7/snapshot.go

```diff
@@ -209,15 +209,15 @@ func (snap *Snapshot) details(id keyID, key string) (interface{}, string, *wirec
 		eval = snap.evaluators[id]
 	}
 	if eval == nil {
-		err := fmt.Errorf("error getting value: value not found for key %s."+
-			" Here are the available keys: %s", key, strings.Join(snap.GetAllKeys(), ","))
-		snap.logger.Errorf("%v", err)
-		return nil, "", nil, nil, err
+		var message = "Failed to evaluate setting '%s' (the key was not found in config JSON). Available keys: %s."
+		var availableKeys = strings.Join(snap.GetAllKeys(), ", ")
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`

Reply:
Done